From 4ff4b104d6ee7690c3ddb6778f578eb4ea2e97ed Mon Sep 17 00:00:00 2001
From: Andrey Grodzovsky <Andrey.Grodzovsky@amd.com>
Date: Tue, 24 Jan 2017 17:28:50 -0500
Subject: [PATCH 0508/2094] drm/amd/display: Rename atomic_commit parameter.

This parameter name is misleading. It's previous
meaning confuses with ASYNC (immidiate flip)
which is totatlly different thing then nonblocking
commit.

Change-Id: Ia8ed1fad025b9b1344a470eff100da9cb1c46d31
Signed-off-by: Andrey Grodzovsky <Andrey.Grodzovsky@amd.com>
Reviewed-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
index a267498..865ee1e 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
@@ -2481,7 +2481,7 @@ static void remove_stream(struct amdgpu_device *adev, struct amdgpu_crtc *acrtc)
 int amdgpu_dm_atomic_commit(
 	struct drm_device *dev,
 	struct drm_atomic_state *state,
-	bool async)
+	bool nonblock)
 {
 	struct amdgpu_device *adev = dev->dev_private;
 	struct amdgpu_display_manager *dm = &adev->dm;
@@ -2510,7 +2510,7 @@ int amdgpu_dm_atomic_commit(
 	 * we should not pin/unpin the fb here, it should be done in
 	 * amdgpu_crtc_flip and from the vblank irq handler.
 	 */
-	if (!async) {
+	if (!nonblock) {
 		ret = drm_atomic_helper_prepare_planes(dev, state);
 		if (ret)
 			return ret;
@@ -2849,7 +2849,7 @@ int amdgpu_dm_atomic_commit(
 	/* In this state all old framebuffers would be unpinned */
 
 	/* TODO: Revisit when we support true asynchronous commit.*/
-	if (!async)
+	if (!nonblock)
 		drm_atomic_helper_cleanup_planes(dev, state);
 
 	drm_atomic_state_free(state);
-- 
2.7.4

